Cargo.toml

```diff
@@ -23,7 +23,9 @@ serde = "1.0.144"
 serde_derive = "1.0.144"
 toml = "0.5.9"
 serde_json = "1.0"
-reqwest = { version = "0.11", features = ["blocking", "json"] } 
+reqwest = { version = "0.11", features = ["blocking", "json"] }
+pyo3 = { version = "0.17.1", features = ["abi3"] }
+openssl = { version = "0.10", features = ["vendored"] } # included to build PyPi Wheels (see .github/workflow/README.md)
```

Comment:
would this be considered a dev dependency?

Reply:
I don't see the README so I'll make this change and if it breaks the action we can resolve for the next release hopefully.

Reply:
actually thinking about it. it might be something we'd need for building with `huak` anyway. so i'll leave it to retain the PR as is, but I'd love your feedback here @dpgraham4401

Reply:
If I remember correctly, I tried it as a dev dependency, but when the wheels get built it gets left out. I was running into this issue https://github.com/messense/maturin-action/discussions/78

Reply:
> I tried it as a dev dependency, but when the wheels get built it gets left out

I figured.